src/Kernel.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
```

Comment:
docheader missing

Reply:
the Kernel is a symfony file. We should not add our docheader to it ;)

Reply:
alright